microprofile.ls/org.eclipse.lsp4mp.ls/src/main/resources/org/eclipse/lsp4mp/snippets/jax-rs.json

```diff
@@ -2,9 +2,7 @@
   "JAX-RS - new resource class": {
     "prefix": "jaxrc",
     "body": [
-      "package ${1:packagename};",
-      "",
-      "import javax.ws.rs.GET;",
+      "${packagename}import javax.ws.rs.GET;",
```

Comment:
I think having the package name variable on a separate line would make more sense if it is possible. To me it wasn't immediately obvious the package name gets expanded on a separate line.

Ie. something like this
```
"body": [
    "${packagename}",
    "import javax.ws.rs.GET;"
    ...
]
```

Reply:
@rzgry I upated my strategy to manage package and now snippets are the same than before.